PSReadLine/Cmdlets.cs

```diff
@@ -237,10 +244,12 @@ public object ContinuationPromptColor
 
         /// <summary>
         /// This handler is called before adding a command line to history.
-        /// The return value indicates if the command line should be added
-        /// to history or not.
+        /// The return value indicates if the command line should be skipped,
+        /// or added to memory only, or added to both memory and history file.
```

Comment:
Are you sure the "memory only" option is that useful?  I think it makes the mental model here harder than it needs to be which, IMO should be simply this - you either save a command to history or you don't.  That mental model makes writing your own custom handler easier too - just return $true/$false - not some enum value I've got to look up.

Reply:
I think it’s useful to provide this option, so command a can be recalled in the same session. This IMO is especially important for the default handler to be less surprising to the user.
In the meantime, it’s still allowed to return true/false from ‘AddToHistoryHandler’, and the behavior is exactly what you want: true == MemoryAndFile, false == SkipAdding.

Reply:
OK, cool $true/$false still works as expected. In that case, this seems reasonable.